### PR TITLE
0.21.0 blog post

### PR DESCRIPTION
**Pull request by devlead**

### CODE REVIEW

File: input/blog/2017-07-14-cake-v0.21.0-released.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Version 0.21.0 of Cake has been released.
+
+So, let's be clear, this release will break things - but it is all intentional!
+
+This release is mostly about bringing the .NET Core aliases up-to-date with .NET Core SDK 1.0.4, which is the latest RTM/stable version.
+The shift to using MSBuild and from `project.json` to `csproj` project system has resulted in many breaking changes and additions to the .NET CLI.
+
+Regardless of intentional or unintentional breaking changes, the possibility of new releases breaking something is why we always recommend you pin the version of Cake you're running, this to avoid any unplanned broken builds. You can read more about pinning Cake version [here](http://cakebuild.net/docs/tutorials/pinning-cake-version).
+
+Huge thanks to [Jonathan Kuleff](https://github.com/JonCubed), who have done a brilliant job and epic effort in methodically going through the .NET CLI and Cake aliases identifying what's been changes, removed or added! It's been great working with you through these pull-requests.
```
Comment (gep13) [inline input/blog/2017-07-14-cake-v0.21.0-released.md]: Who has done...

Comment (gep13) [inline input/blog/2017-07-14-cake-v0.21.0-released.md]: What's been changed...

File: input/blog/2017-07-14-cake-v0.21.0-released.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+Version 0.21.0 of Cake has been released.
+
+So, let's be clear, this release will break things - but it is all intentional!
+
+This release is mostly about bringing the .NET Core aliases up-to-date with .NET Core SDK 1.0.4, which is the latest RTM/stable version.
+The shift to using MSBuild and from `project.json` to `csproj` project system has resulted in many breaking changes and additions to the .NET CLI.
+
+Regardless of intentional or unintentional breaking changes, the possibility of new releases breaking something is why we always recommend you pin the version of Cake you're running, this to avoid any unplanned broken builds. You can read more about pinning Cake version [here](http://cakebuild.net/docs/tutorials/pinning-cake-version).
```
Comment (gep13) [inline input/blog/2017-07-14-cake-v0.21.0-released.md]: Thus avoiding...

Comment (devlead) [inline input/blog/2017-07-14-cake-v0.21.0-released.md]: changed

File: input/blog/2017-07-14-cake-v0.21.0-released.md (hunk 1 above)
Comment (devlead) [inline input/blog/2017-07-14-cake-v0.21.0-released.md]: changed

**Pull request merged**